tools/releaser/breaking_changes/v20240805004.md

```diff
@@ -2,7 +2,7 @@
 
 ## SDK changes
 
-go.mongodb.org/atlas-sdk/v20240805004/admin
```

Comment:
This looks unintentional

Reply:
that's being done in all releases, but you're right we shouldn't need to change breaking changes files, create PR to fix this, but not a blocker for this PR: CLOUDP-279233